modin/pandas/test/test_series.py

```diff
@@ -1533,12 +1537,17 @@ def test_dropna_inplace(data):
     df_equals(modin_series, pandas_series)
 
 
-@pytest.mark.xfail_backends(
-    ["BaseOnPython"], reason="Empty Series has a missmatched from Pandas dtype."
-)
 def test_dtype_empty():
     modin_series, pandas_series = pd.Series(), pandas.Series()
-    assert modin_series.dtype == pandas_series.dtype
+    try:
+        assert modin_series.dtype == pandas_series.dtype
+    except Exception as e:
+        # TODO delete this except when future version Pandas will be released (Issue #3142)
+        if get_current_backend() == "BaseOnPython":
+            with pytest.raises(AssertionError):
+                assert modin_series.dtype == pandas_series.dtype
+        else:
+            raise e
```

Comment:
I would highlight the difference between modin and pandas dtype:
```suggestion
    except AssertionError as e:
        # TODO delete this except when future version Pandas will be released (Issue #3142)
        if get_current_backend() == "BaseOnPython":
            assert modin_series.dtype == np.object
            assert pandas_series.dtype == np.float64
        else:
            raise e
```